### DISCUSSION TITLE
自动同步后，obsidian中的图片url变成了本地地址

### DISCUSSION THREAD

**Discussion by Cha5elx** [windingwind/zotero-better-notes#1176] [Q&A]
我在zotero使用better-notes导出笔记后，使用obsidian在笔记里添加了图片，我的图片是上传到图床里去的，所以是url的形式。但是在better-notes自动同步之后，图片url就变成了本地的地址，图片保存在了附件文件夹里，这个问题要怎么解决？
![image](https://github.com/user-attachments/assets/3f8a2ec2-d4d0-4cf0-80ce-1077d0c7d6a3)
![image](https://github.com/user-attachments/assets/53b39327-0a9f-4515-b98b-3ba60a642eac)

**Comment by windingwind** [comment]
zotero不支持图床，因此自动同步后会变成本地图片
